<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="mainfield">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="acceptRichText">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Изменить цвет...</string>
     </property>
     <addaction name="actionBackColor"/>
     <addaction name="actionFontColor"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="actionFont"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionGitHub"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Изменить шрифт</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Фона</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Текста</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Изменить размер</string>
   </property>
  </action>
  <action name="actionBackColor">
   <property name="text">
    <string>Фона</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="text">
    <string>Текста</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Изменить шрифт</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>GitHub разработчика</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Изображение</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
